movingpandas/trajectory_collection.py

```diff
@@ -66,6 +66,19 @@ def __iter__(self):
                 raise ValueError(f"Trajectory with length >= 2 expected: "
                                  f"got length {len(traj.df)}")
 
+    def copy(self):
```

Comment:
I have left this in as I thought it might be useful. It can be removed if deemed undesirable.